backend/utils_segment_whole_slide.py

```diff
@@ -222,7 +231,7 @@ def tile_and_save_contours(
     mpp_y: float,
     saved_contours,
     downsample_factor: float,
-    tile_size: int = 512,
+    tile_size:  TILE_SIZE,
```

Comment:
Incorrect: TILE_SIZE is not a type!
```suggestion
    tile_size:  int = TILE_SIZE
```

Reply:
Changed back to int.